src/components/Feed/Events/GalleryUpdatedFeedEvent.tsx

```diff
@@ -0,0 +1,176 @@
+    // @ts-expect-error wait multigallery to be merged
+    pathname: '[username]/galleries/[galleryId]',
+    query: {
+      username: event.owner.username,
+      galleryId: event.gallery.dbid,
+    },
+  };
+
+  return (
+    <StyledEvent>
+      <VStack gap={16}>
+        <StyledEventHeader>
+          <HStack gap={4} inline>
+            <BaseM>
+              <HoverCardOnUsername userRef={event.owner} queryRef={query} /> updated{' '}
+              <InteractiveLink to={galleryPagePath}>
+                {event?.gallery?.name || 'gallery'}
+              </InteractiveLink>
+            </BaseM>
+            <StyledTime>{getTimeSince(event.eventTime)}</StyledTime>
+          </HStack>
+        </StyledEventHeader>
+        <StyledEventContent>
+          {caption && (
+            <StyledCaptionContainer gap={8} align="center">
+              <BaseM>{caption}</BaseM>
+            </StyledCaptionContainer>
+          )}
+          {event?.subEventDatas?.map((subEvent, index) => {
+            switch (subEvent.__typename) {
+              case 'CollectionCreatedFeedEventData':
+                return (
+                  <CollectionCreatedFeedEvent
+                    key={index}
+                    eventDataRef={subEvent}
+                    queryRef={query}
+                    caption={null}
+                    isSubEvent
+                  />
+                );
+              case 'CollectionUpdatedFeedEventData':
+                return (
+                  <CollectionUpdatedFeedEvent eventDataRef={subEvent} queryRef={query} isSubEvent />
+                );
+              case 'CollectorsNoteAddedToTokenFeedEventData':
+                return (
+                  <CollectorsNoteAddedToTokenFeedEvent
+                    eventDataRef={subEvent}
+                    queryRef={query}
+                    isSubEvent
+                  />
+                );
+              case 'TokensAddedToCollectionFeedEventData':
+                return (
+                  <TokensAddedToCollectionFeedEvent
+                    caption={null}
+                    eventDataRef={subEvent}
+                    queryRef={query}
+                    isSubEvent
+                  />
+                );
+              case 'CollectorsNoteAddedToCollectionFeedEventData':
+                return (
+                  <CollectorsNoteAddedToCollectionFeedEvent
+                    eventDataRef={subEvent}
+                    queryRef={query}
+                    isSubEvent
+                  />
+                );
+              case 'UserFollowedUsersFeedEventData':
+                return (
+                  <UserFollowedUsersFeedEvent
+                    eventDataRef={subEvent}
+                    queryRef={query}
+                    feedMode={feedMode}
+                    isSubEvent
+                  />
+                );
+              default:
+                throw new TriedToRenderUnsupportedFeedEvent(subEvent.__typename);
```

Comment:
I think we can reuse the `<FeedEvent/>` component here instead of duplicate code. 

Will revisit again when the backend issue solved.